ls1intum/Artemis
# `Communication`: Increase the size of the question screen

### Checklist
#### General


#### Client

### Motivation and Context
The current container in which posts are listed violates our design principle (as implemented in the Exercises, Lectures and Statistics Tab) that the whole page should scroll. Solves #5906.

### Description
We remove the container for posts and instead allow the user to scroll the whole page to browse course posts.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Course with communication features enabled

1. Log in to Artemis
2. Navigate to Course Couumunication Overview
3. Make sure that posts are not displayed within a container and that the whole page can be scrolled
4. Make sure that the posts scroll without disruptions
5. Open your browser's developer tab and verify that items on the DOM tree are updated instead of appending new items as the user scrolls the page.

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage




Not changed.

### Screenshots

https://user-images.githubusercontent.com/67344158/208880472-27035f20-667c-4e61-b90a-a15ae2448f96.mp4
